app/views/layouts/application.html.haml

```diff
@@ -26,11 +26,12 @@
     %title= html_title
 
     = theme_style_tags current_theme
+    = vite_client_tag
+    = vite_react_refresh_tag
     -# Needed for the wicg-inert polyfill. It needs to be on it's own <style> tag, with this `id`
-    = stylesheet_pack_tag 'inert', media: 'all', crossorigin: 'anonymous', id: 'inert-style'
+    = vite_stylesheet_tag 'styles/inert.scss', media: 'all', id: 'inert-style'
 
-    = javascript_pack_tag 'common', crossorigin: 'anonymous'
```

Comment:
We should ensure that `common` did not execute any JS side effect, or load any CSS (I dont think I checked this)

Reply:
`common` as in `common.js`? the only thing that file does is start the Rails js file.

Reply:
I think that `rails-ujs` runs some side effects, so we need to ensure that it is imported on every page, but as think that its the case because we call `start()` in `application.ts`, and this file is included everywhere `common` was previously called?

… and after checking, the `common` chunk was special in the webpack configuration, and was the default chunk. So as long as we still call `start()` in every "main" entrypoint (the ones that are loaded from Rails layouts so that we have at least one of those on every page), we should be good

Reply:
so right now it's imported by `application.ts`, `embed.tsx`, `public.tsx`, and `share.tsx`. i'm not sure what you mean by "main". here are the ones that don't import it that i can see:

* `admin.tsx`
* `error.ts`
* `remote_application_helper.ts`
* `sign_up.tsx`
* `two_factor_authentication.ts`

Reply:
fixed with c7b039ce81a00fd982ef8057fa8f8e9e4c9abeb4. i imported common everywhere, but perhaps it'd be fine to rely on just `application.html.haml`?